src/java/org/jivesoftware/openfire/crowd/CrowdAuthProvider.java

```diff
@@ -47,9 +47,6 @@ public CrowdAuthProvider() {
      * Returns if the username and password are valid; otherwise this
      * method throws an UnauthorizedException.<p>
      *
-     * If {@link #isPlainSupported()} returns false, this method should
-     * throw an UnsupportedOperationException.
-     *
```

Comment:
Yes, incorrect, method doesn't throw `UnsupportedOperationException`, nor is isPlainSupported() a method.